tests/functional/olp-cpp-sdk-dataservice-write/DataserviceWriteVersionedLayerClientTest.cpp

```diff
@@ -40,6 +40,7 @@ const std::string kLayer = "layer";
 const std::string kLayer2 = "layer2";
 const std::string kLayerSdii = "layer_sdii";
 const std::string kVersionedLayer = "versioned_layer";
+const auto kWaitBeforeRetry = std::chrono::seconds(6);
```

Comment:
Please, add comment on this constant about why it was

Reply:
Done